README.md

```diff
@@ -282,20 +282,16 @@ Examples:
    odxtools browse ./path/to/database.pdx
 
 positional arguments:
-  {list,browse,snoop,find,encode-message,decode-message}
+  {list,browse,snoop,find}
                         Select a sub command
     list                Print a summary of automotive diagnostic files.
     browse              Interactively browse the content of automotive diagnostic files.
     snoop               Live decoding of a diagnostic session.
     find                Find & display services by hex-data, or name, can also decodes requests.
-    encode-message      Encode a message. Interactively asks for parameter values.
-                        This is a short cut through the browse command to directly encode a message.
-    decode-message      Decode a message. Interactively asks for parameter values.
-                        This is a short cut through the browse command to directly encode a message.
 
```

Comment:
thanks for all the README updates. I think I went over it too many times to notice mistakes ;)

not to myself: the `encode-message` and `decode-message` tools seem to have been removed by accident in #1. (this is a reason why PRs should be small and self contained.) TODO: investigate if they can/should be brought back.